views/components/PromptCard.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import styled from 'styled-components';
 
 const CardWrapper = styled.div`
-  box-shadow: 0 2px 3px rgba(0,0,0,0.2);
-  padding: 8px;
+  box-shadow: var(--ds-shadow-raised,0 1px 1px rgba(9,30,66,0.25),0 0 1px rgba(9,30,66,0.31));
```

Comment:
For the sake of knowledge sharing, the recommendation of using *design token* can be found here: https://atlassian.design/components/stylelint-design-system/usage

- Introduction of design tokens: https://atlassian.design/tokens/design-tokens
- A list of deign tokens: https://atlassian.design/components/tokens/all-tokens